IND/MarketIndices/BasketIndices.rdf

```diff
@@ -135,7 +136,7 @@
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">The constituents of the basket, which is a collection of risks, are defined as a part of the contract. So this not an index of contracts but of credit events as they affect a notional portfolio. If the index changes there may be partial payouts, but the contract continues in existence. Also, such an index does not necessarily reference a static portfolio, as there may be proivisions for replacing defaulted securities. Premiums are standard and set depending on the composite spread at the initiation of the index, with upfront-fees to get into a deal thereafter depending on changes in the MTM of the spread. The payout per constituent credit-event is roughly Notional x weight. The constituent is removed, the Notional*weight is paid, the remaining notional lowered, and the weighting re-balanced. Notionals can be large but the payouts are small because of the small weights. For example, $250MM Notional, 125 entities, weight is 1/125 for a defaulting entity = $2MM (simplified) transfer.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
Unclear what's meant by "the contract" in the first sentence. Overall this seems to confuse an Index with an investment contract based on that index. Just like an investment pegged to the Consumer Price Index is distinct from the Index.

Reply:
I've removed elements of the note that reflect a contract that depends on the index, and only retained information about the index itself.